code/game/gamemodes/events/holidays/Easter.dm

```diff
@@ -1 +1,17 @@
-//placeholder for holiday stuff
\ No newline at end of file
+/proc/Easter_Game_Start()
+    Random_Egg()
+
+//Random egg
+/proc/Random_Egg()
+    world << "<h3>There is a golden egg hidden somewhere on the station...</h3>"
+    /var/list/turf/simulated/floor/Floorlist = list()
+    for(var/turf/simulated/floor/T)
```

Comment:
While this is technically valid, it should be using the `turfs` list instead of trawling world.